lib/kino_db/connection_cell.ex

```diff
@@ -127,6 +133,35 @@ defmodule KinoDB.ConnectionCell do
     end
   end
 
+  defp to_quoted(%{"type" => "bigquery"} = attrs) do
+    goth_source_type =
+      if type = attrs["credentials"]["type"],
+        do: String.to_atom(type),
```

Comment:
This is incorrect. If you authenticate using `$ gcloud auth login`, you'd get a `~/.config/gcloud/application_default_credentials.json` that contains credentials with a refresh token but type "authorized_user", not "refresh_token". Don't dynamically generate atom from string, pattern match on the string. If you don't intend to support anything besides service account, which is fine because we're just getting started, please do so explicitly in code.

Reply:
Yeah, I've added a TODO comment to support `:refresh_token` and `:metadata` in the future.